### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by raisedadead**
Adding the standard Github templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (ojongerius) [inline .github/ISSUE_TEMPLATE/Bug_report.md:23]: This is irrelevant.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
```
Comment (ojongerius) [inline .github/ISSUE_TEMPLATE/Bug_report.md:13]: Probably not the way to reproduce a graphql bug 😉

**Comment by raisedadead** [comment]
Oh yeah.. you are correct I was testing the templates and was supposed to make them relevant, before I got swamped. I'll make PR when I get to it.

**Comment by raisedadead** [comment]
https://github.com/freeCodeCamp/open-api/issues/121

**Comment by ojongerius** [comment]
No worries, if raised a PR in #122

**Pull request merged**